backend/routes/profile.js

```diff
@@ -20,16 +20,16 @@ router.get('/', async (req, res) => {
   }
 });
 
-router.post('/', verifyToken, async (req, res) => {
+router.put('/', verifyToken, async (req, res) => {
   const {
-    name, profilePicture, bio, phone, address, city, state, school, course, interest
+    name, profilePicture, bio,email, phone, address, city, state, school, major, interest, mentorship
```

Comment:
If we're allowing users to update their email, would we re-send a verification email to the new email?

Reply:
I have made changes to this so the user can not edit that part. Thanks for pointing me to this.